package.json

```diff
@@ -20,15 +20,22 @@
     "not ie < 11",
     "not IE_Mob 11"
   ],
-  "main": "./target/js/okta-sign-in.js",
   "types": "./types/index.d.ts",
+  "main": "./target/js/okta-sign-in.js",
+  "module": "./target/esm/src/index.js",
+  "exports": {
+    "require": "./target/js/okta-sign-in.js",
+    "import": "./target/esm/src/index.js"
```

Comment:
nit: also add `"./package.json": "./package.json"`, so it can be accessed externally

Reply:
<img width="428" alt="Screen Shot 2022-04-25 at 4 27 39 PM" src="https://user-images.githubusercontent.com/47947204/165190774-7546fb26-6b0f-4308-b296-8090d417d657.png">

Seeing this error in editor, but not sure where it is coming from. Lint seems to be fine.

Reply:
Got this in downstream:

```
Error [ERR_INVALID_PACKAGE_CONFIG]: Invalid package config /root/okta/okta-auth-js/node_modules/@okta/okta-signin-widget/package.json. "exports" cannot contain some keys starting with '.' and some not. The exports object must either be an object of package subpath keys or an object of main entry condition name keys only.
```

Reply:
what about 
```
exports: {
  ".": {
    require: ...,
    import: ...
  },
  "./package.json": "./package.json"
}